autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/ambari/AmbariTotalHostsEvaluator.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.periscope.monitor.evaluator.ambari;
+
+import com.sequenceiq.ambari.client.AmbariClient;
+import com.sequenceiq.periscope.aspects.RequestLogging;
+import com.sequenceiq.periscope.domain.Cluster;
+import com.sequenceiq.periscope.domain.ClusterManagerVariant;
+import com.sequenceiq.periscope.monitor.evaluator.ClusterManagerTotalHostsEvaluator;
+import com.sequenceiq.periscope.service.AmbariClientProvider;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component("AmbariTotalHostsEvaluator")
+@Scope("prototype")
+public class AmbariTotalHostsEvaluator implements ClusterManagerTotalHostsEvaluator {
```

Comment:
Do we need Ambari implementation?

Reply:
Do you mean we don't need to support Ambari anymore here?

Reply:
Yep, we cleaned the Ambari from Cloudbreak. I think we don't need to support in Periscope as well. @keyki what do you think?

Reply:
Yes, we don't need Ambari anymore.